src/main/java/me/botsko/prism/Prism.java

```diff
@@ -653,56 +667,22 @@ public void alertPlayers(Player player, String msg) {
     }
 
     /**
-     * Inform the player of missing arguments
-     *
-     * @return
-     */
-    public String msgMissingArguments() {
-        return messenger.playerError("Missing arguments. Check /prism ? for help.");
-    }
-
-    /**
-     * Inform the player of invalid arguments
+     * Report nearby players of a set radius.
      *
-     * @return
-     */
-    public String msgInvalidArguments() {
-        return messenger.playerError("Invalid arguments. Check /prism ? for help.");
-    }
-
-    /**
-     * Inform the player of an invalid command
-     *
-     * @return
-     */
-    public String msgInvalidSubcommand() {
-        return messenger.playerError("Prism doesn't have that command. Check /prism ? for help.");
-    }
-
-    /**
-     * Inform the player of a missing permission
-     *
-     * @return
-     */
-    public String msgNoPermission() {
-        return messenger.playerError("You don't have permission to perform this action.");
-    }
-
-    /**
-     * Report nearby players
-     *
-     * @param player
-     * @param msg
+     * @param player Player
+     * @param radius int
+     * @param msg    String
      */
     public void notifyNearby(Player player, int radius, String msg) {
         if (!getConfig().getBoolean("prism.appliers.notify-nearby.enabled")) {
             return;
         }
+        int distance = (radius
+                + config.getInt("prism.appliers.notify-nearby.additional-radius")) ^ 2;
         for (final Player p : player.getServer().getOnlinePlayers()) {
-            if (!p.equals(player)) {
+            if (!p.getUniqueId().equals(player.getUniqueId())) {
                 if (player.getWorld().equals(p.getWorld())) {
-                    if (player.getLocation().distance(p.getLocation()) <= (radius
-                          + config.getInt("prism.appliers.notify-nearby.additional-radius"))) {
+                    if (player.getLocation().distanceSquared(p.getLocation()) <= distance) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)